pyanaconda/modules/timezone/installation.py

```diff
@@ -22,7 +22,7 @@
 from pyanaconda.modules.common.constants.services import TIMEZONE
 from pyanaconda.anaconda_loggers import get_module_logger
```

Comment:
You have a typo in the commit message 5fe34f7afda7deb86929358774fabb724a761556. `alraedy`

Reply:
Solved as f09b29868fd8d3be61610e86f110423a0d5ef452